README.md

```diff
@@ -161,7 +162,13 @@ See the later topic, *Install Intel(R) SGX PSW*, for information on how to insta
   $ make
   $ ./app
 ```
-   Use similar commands for other code samples.   
+   Use similar commands for other code samples.
+
+   NOTE: You may need to set up the needed environment variables before compiling the code sample. To do so, run:
+```
+   $ source ../../environment
```

Comment:
User can install the sdk to other places, so we'd use ${sgx-sdk-install-path} here. Same comments for the other use of the relative path of 'environment'